index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="style.css" />
+		<title>Document</title>
+	</head>
+	<body>
+		<header>
+			<nav class="nav__menu">
+				<figure class="nav__logo">
+					<img src="./images/logo.svg" alt="logo" />
+				</figure>
+				<ul class="nav__list">
+					<li class="nav__item"><a href="#" class="nav__link">The Team</a></li>
+					<li class="nav__item"><a href="#" class="nav__link">Pricing</a></li>
+					<li class="nav__item"><a href="#" class="nav__link">Features</a></li>
+				</ul>
+			</nav>
+			<div class="header__content">
+				<div class="header__content__info">
+					<h1 class="header__heading">Beautiful Free Nova<br />template</h1>
+					<p class="header__paragraph">
+						A top notch premium quality <br />
+						template for your next web <br />
+						project.
+					</p>
+					<button class="header__btn">Download <span>free!</span></button>
+				</div>
+				<figure class="header__img">
+					<img src="./images/screen.png" alt="screen image" />
+				</figure>
+			</div>
+		</header>
+		<main>
+			<section class="features__content">
+				<div class="features__container">
```

Comment:
Można też użyć znacznika header` - może to być nagłówek całej strony, ale też w sekcji czy artykule :)